nuxeo-core/nuxeo-core-storage-dbs/src/main/java/org/nuxeo/ecm/core/storage/dbs/DBSSession.java

```diff
@@ -175,6 +178,7 @@ public DBSSession(DBSRepository repository) {
         saveTimer = registry.timer(MetricRegistry.name("nuxeo", "repositories", repository.getName(), "saves"));
         queryTimer = registry.timer(MetricRegistry.name("nuxeo", "repositories", repository.getName(), "queries"));
         LOG_MIN_DURATION_NS = Long.parseLong(Framework.getProperty(LOG_MIN_DURATION_KEY, "-1")) * 1000000;
+        isLatestVersionDisabled = Boolean.parseBoolean(Framework.getProperty(DISABLED_ISLATESTVERSION_PROPERTY, "false"));
```

Comment:
Guess you could use `Framework#isBooleanPropertyTrue(DISABLED_ISLATESTVERSION_PROPERTY)` here?